backend/src/middleware/permissionsMiddleware.spec.js

```diff
@@ -40,10 +40,13 @@ describe('authorization', () => {
         })
 
         it('does not expose the owner\'s email address', async () => {
+          let response = {}
           try {
             await action()
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-02T17:39:27Z" title="Thursday, May 2nd 2019, 7:39:27 pm +02:00">May 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -43,7 +43,8 @@ describe('authorization', () => {
           try {
             await action()
           } catch (error) {
-            expect(error.response.data).toEqual({ User: [ { email: null } ] })
+            console.log(error.response.data)
+            expect(error.response.data).toEqual({ User: [ null ] })
```


Why this change? :thinking:

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-05-03T10:02:00Z" title="Friday, May 3rd 2019, 12:02:00 pm +02:00">May 3, 2019</time>_

---


To pass the test

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-06T16:53:50Z" title="Monday, May 6th 2019, 6:53:50 pm +02:00">May 6, 2019</time>_

---


We found the reason for it: If you have an exclamation mark in the schema e.g. `email!` and the response would e.g. expect the email for each `User` object, and the server does not have an email for the client, then the entire `User` object gets nullified.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-06T16:54:11Z" title="Monday, May 6th 2019, 6:54:11 pm +02:00">May 6, 2019</time>_

---


@mattwr18 @appinteractive @Tirokk :point_up: